File: 26. Right Triangle TL Even Numbers/tlen.py
n = int(input("Enter a Number : "))

for i in range(1,n+1):
    for j in range(1,i+1):
        print(2*j, end = " ")
    print()

'''
Output:

2 
2 4 
2 4 6 
2 4 6 8 
2 4 6 8 10    
'''

File: Patterns/3. Right Triangle Patterns (Top-Left)/26. Right Triangle TL Even Numbers/tlen.cpp
#include <iostream>
using namespace std;
int main(){
    int n;
    cout<<"Enter a Number : ";
    cin>>n;

    for(int i=1; i<=n; i++){
        for (int j=1; j<=i; j++){
                cout<<2*j<<" ";
        }
        cout << '\n';
    }
}

/* Output

2 
2 4 
2 4 6 
2 4 6 8 
2 4 6 8 10   

*/ 